scripts/new-package.js

```diff
@@ -0,0 +1,7 @@
+const shell = require("child_process").execSync;
+const { join } = require("path");
+const packageName = process.argv.pop();
+const src = join(__dirname, "../template");
+const dest = join(__dirname, "../packages", packageName);
+shell(`mkdir -p ${dest}`);
```

Comment:
Maybe we should expend a few more lines to make this work in Windows, too. fs has a mkdir and a copyFile API; used with path to take care of different separators, this should be fairly easy.

Reply:
Yeah good idea.
Just to be clear (I'm not that familiar with node), are you proposing to use `path` + `fs` to copy files, instead of executing shell commands? I could try doing that.. also maybe do some search and replace on the readme and package.json, so that this helper is actually doing something useful besides copy-pasting :)

Reply:
ok done, node is surprisingly easy, wow